uidefinitions/createUiDefinition.json

```diff
@@ -1220,7 +1216,7 @@
                 "name": "maxNodes",
                 "type": "Microsoft.Common.TextBox",
                 "label": "Max number of nodes",
-                "defaultValue": "1",
+                "defaultValue": "100",
```

Comment:
This default value is pretty large for most of customers. I would reduce this to something like 32